tagger.go

```diff
@@ -238,6 +285,12 @@ func main() {
 		}).Error("Failed to apply new tag sets to instances")
 	}
 
+	// set report status
+	reportFlag := viper.GetString("report")
+	if reportFlag != "" {
```

Comment:
```suggestion
	if viper.GetBool("report") {
```

The report flag is a boolean, let's use it like one!